web/source/keymanweb.js

```diff
@@ -2817,6 +2817,25 @@ if(!window['keyman']['initialized']) {
     if (window.addEventListener)
       window.addEventListener('focus', keymanweb._BubbledFocus, true);  
 
+    /**
+     * Function     _GetEventKeyCode
+     * Scope        Private
+     * @param       {Event}       e         Event object
+     * Description  Finds the key code represented by the event, prioritizing modern methods over deprecated ones.
+     *              Necessary because the 'modern' methods aren't yet fully implemented across all browsers at this time.
+     */
+    keymanweb._GetEventKeyCode = function(e) {
+      if(e.code && keymanweb._codeToLcodeMap[e.code]) {  // Unfortunately, the former DOES NOT map to a nice integer code!
```

Comment:
I'm not in favour of using `event.code` at present. Support is still inconsistent (IE, Edge, Android Browser, Chrome on Android don't support it - https://caniuse.com/#feat=keyboardevent-code) and I can't see that it actually adds any benefit at this time -- instead bulking out our deployable code with a big map of strings.

Reply:
I've tried searching for documentation on IE, but to no avail.  It's a shame that link couldn't provide a reference link to that 'inconsistent' statement for us.